server/datastore/sql/migrations/001-initial.sql

```diff
@@ -15,3 +15,21 @@ CREATE TABLE posts (
   postedAt INTEGER NOT NULL,
   FOREIGN KEY (userId) REFERENCES users (id)
 );
+
+CREATE Table comments (
```

Comment:
you'll want to add a second migration file, like `002-add-comments-and-likes.sql` for example, with these changes.
the point of database migration is that they should never be re-run, and all schema updates should be done incrementally to keep backward compatibility. let me know if you wanna discuss more here. :)

Reply:
resolved.